src/System.Drawing.Common/src/System/Drawing/Bitmap.Serializable.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Drawing
+{
+    using System.Runtime.Serialization;
```

Comment:
As long as you're moving stuff to new files, could you format things like we do elsewhere, e.g. usings at the top outside of any namespaces?